clustering/cluster_minifigures.py

```diff
@@ -0,0 +1,34 @@
+def list_categories(file):
+    with open(file) as minifigures:
+        categories_list = []
+        for line in minifigures:
+            if 'Category ID' in line:
+                continue
+            content = line.strip().split('\t')
+            if len(content) < 2:
+                continue
+            category_id = content[0]
+            if category_id not in categories_list:
+                categories_list.append(category_id)
+    return categories_list
+
+categories = list_categories('minifigures.txt')
+
+with open('categories.json', 'w') as json_file:
```

Comment:
i would move write to file move to an own method like

`def write_to_json(data, file):
        with open(file, 'w') as json_file:
              json_file.write(json.dumps(data))
`